test/e2e/node_reuse.go

```diff
@@ -90,7 +90,7 @@ func nodeReuse(ctx context.Context, inputGetter func() NodeReuseInput) {
 	m3machineTemplateName := fmt.Sprintf("%s-controlplane", input.ClusterName)
 	updateNodeReuse(ctx, input.Namespace, true, m3machineTemplateName, managementClusterClient)
 	newM3machineTemplateName := fmt.Sprintf("%s-new-controlplane", input.ClusterName)
-	createNewM3machineTemplate(ctx, input.Namespace, newM3machineTemplateName, m3machineTemplateName, managementClusterClient, imageURL, imageChecksum, "raw", "md5")
+	createNewM3machineTemplate(ctx, input.Namespace, newM3machineTemplateName, m3machineTemplateName, managementClusterClient, imageURL, imageChecksum, "sha256", "raw")
```

Comment:
How did this work before? :flushed:

Reply:
It worked because the variables were mixed on L491 and L492 ... ouch.

Reply:
Thanks I thought I was going crazy! :joy:

Reply:
I had the same reaction when I saw it. 😂